lib/brisket.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+require("./polyfill")();
```

Comment:
I do. I don't know which version of Node.js the user is using. It is a negligible start-up time perf hit in almost all if not all cases anyways.